From e373c2400657386cc9f70bedb76900d45123f855 Mon Sep 17 00:00:00 2001
From: Stefano Karapetsas <stefano@karapetsas.com>
Date: Tue, 11 Dec 2012 19:51:17 +0000
Subject: add gio-unix-2.0 to gio requirements

it fixes "main.c:26:33: fatal error: gio/gdesktopappinfo.h: No such file or directory"
---
diff --git a/configure.in b/configure.in
index 44bfdc8..c67621c 100644
--- a/configure.in
+++ b/configure.in
@@ -102,7 +102,7 @@ dnl -- Check for mate-icon-theme (required) ----------------------------------
 PKG_CHECK_MODULES(GIT, mate-icon-theme >= $MATE_ICON_THEME_REQUIRED)
 
 dnl -- check for gio (required) ------------------------------------------
-PKG_CHECK_MODULES(GIO, gio-2.0 >= $GIO_REQUIRED)
+PKG_CHECK_MODULES(GIO, gio-2.0 >= $GIO_REQUIRED gio-unix-2.0)
 AC_SUBST(GIO_CFLAGS)
 AC_SUBST(GIO_LIBS)
 
--
cgit v0.9.0.3-65-g4555
